dart/lib/src/sentry_browser_client.dart

```diff
@@ -25,13 +23,10 @@ class SentryBrowserClient extends SentryClient {
   ///
   /// If [httpClient] is provided, it is used instead of the default client to
   /// make HTTP calls to Sentry.io. This is useful in tests.
-  factory SentryBrowserClient(SentryOptions options) {
-    options.sdk ??= Sdk(name: sdkName, version: sdkVersion);
-
-    // origin is necessary for sentry to resolve stacktrace
-    return SentryBrowserClient._(options);
-  }
+  factory SentryBrowserClient(SentryOptions options) =>
```

Comment:
make me wonder if now we need to have a SentryBrowserClient and IOClient only because of the `origin` field.

Reply:
true, we could move the origin settings in transport, and use the same strategy than for gzip compression => it will remove the need of 2 clients